portfolio/src/main/webapp/index.html

```diff
@@ -6,80 +6,103 @@
-        </div>
-        <h1 id="intro">Hi! I'm Blessing (also go by Bibie) <br /> welcome to my portfolio</h1>
-        <div id="pics-bottom">
-            <img src="images/pic3.JPG" class="pic" />
-            <img src="images/pic4.jpg" class="pic" />
+    <!-- Home Section -->
+    <section class="home-section" id="home-section">
+        <div class="max-width">
+            <div class="content">
+                <div class="text1">Hello, my name is</div>
+                <div class="text2">Blessing Babajide (Bibie)</div>
+                <div class="text3">Welcome to my Portfolio</div>
+            </div>
         </div>
     </section>
 
-    <section id="about-me">
-        <div class="write-up">
-            <p> I'm originally from Nigeria, a country in West Africa.
-                I have 1 older sister also in college, and 2 little siblings back home.
-                I'm 5 feet tall but i've been told my personality is towering.
-                <hr>
-                I have no pets but I have 2 huge stuffed animals: Bobo a teddy bear and
-                Sparkles a rare unicorn-pegasus offspring,
-                and a bunch of little ones: Nitro a fox, Frosty a snowman, Snow a snowbear,
-                Elvis a Narwhal and Duo (yes from Duolingo).
-                <hr>
-                In my free time, I like to read all sorts of books, write/watch slam poetry,
-                continuously attempt to play the piano, and catch up on sleep!
-            </p>
-        </div>
-        <div class="write-up">
-            <p> I currently go to the University of Rochester in Rochester, NY where
-                I'm a double major in Computer Science and Brain and Cognitive Sciences with a minor in Psychology
-                <hr>
-                I'm very active on campus because I like to keep busy and believe college shouldnt be just about
-                academics.
-                I'm a member of the Google DSC Tech Team, Womanist club, SHPE, and also a leader at our amazing poetry
-                club
-                called No Disclaimers.
-                <hr>
-                Apart from my extra-curriculars, I work as a Teaching Assistant to a CS Intro to Python and I'm a tutor
-                for
-                the same class in addition to intro to psychology, statistics and a class on foundations of behavior.
-                <hr>
-                Google SPS is the closest was the first opportunity I ever had that was closest to a technical
-                internship (I've had research internships before)
-                I' very excited to learn about frontend and backend of building a web - app and also getting to talk to
-                Googlers!
+    <!-- About me Section -->
+    <section class="about-section" id="about-section">
+        <div class="max-width">
+            <h2 class="title">About Me</h2>
+            <div class="about-content">
+                <div class="left"></div>
```

Comment:
just for spacing?

Reply:
yes i couldnt figure out how to space is how i wanted without having a left then right div😅😅 i'm horrible at CSS numbers